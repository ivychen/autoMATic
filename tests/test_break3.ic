/* autoMATic Compiler - Spring 2018 - COMS4115 PLT
 by Ivy Chen ic2389, Nguyen Chi Dung ncd2118,
 Nelson Gomez ng2573, Jimmy O'Donnell jo2474 */

int main() {
	int i;
	int j;
	int k;

	for (i = 0; i < 5; i = i + 1) {
		for (j = 0; j < 5; j = j + 1) {
			for (k = 0; k < 5; k = k + 1) {
				if (k == 5) break 2;
			}
		}

		println(k);
		if (i == 4) break;
	}

	for (i = 10; i < 15; i = i + 1) {
		if (i == 12) continue;
		println(i);
		if (i == 14) break;
	}

	return 0;
}
