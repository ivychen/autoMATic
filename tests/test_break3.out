5
5
5
5
5
10
11
13
14
